robottelo.properties.sample

```diff
@@ -188,3 +188,12 @@ ssh_key=
 # Parameter for deciding whether conduct initial sync or resync
 # 'resync' denotes resync; 'sync' denotes initial sync
 # sync_type='sync'
+
+# [libvirt_host]
```

Comment:
ok, if using separate section helps isolate things then that's good. Ok, I agree separating things would quickly help us know what is required and what is not , for a particular CR.

Also sometimes certain CR's have multiple variables too.